forge/test/mlir/test_ops.py

```diff
@@ -409,16 +409,11 @@ def forward(self, x, y):
     co_out = [co.to("cpu") for co in co_out]
     assert compare_with_golden_pcc(golden=fw_out, calculated=co_out[0], pcc=0.99)
 
-
+@pytest.mark.xfail(reason="Unable to reshape a tensor in TILE_LAYOUT to non-tile height and width! Please convert the tensor to ROW_MAJOR_LAYOUT first")
 @pytest.mark.parametrize("x_shape", [7, 32, 41])
 @pytest.mark.parametrize("y_shape", [7, 32, 41])
 @pytest.mark.parametrize("dim", [1, 2])
 def test_mean(x_shape, y_shape, dim):
-    if dim == 1:
-        pytest.skip("FFE: Unsupported squeeze operation")
-    if dim == 2:
-        # Note: Some tests are passing when run in group, while failing when running individually
-        pytest.skip("TTNN: Tensor layout bugs")
```

Comment:
@nvukobratTT This test are failing with `Unable to reshape a tensor in TILE_LAYOUT to non-tile height and width! Please convert the tensor to ROW_MAJOR_LAYOUT first` issue now.
At dim = -2, earlier it was failed with  TTNN: Tensor layout bugs. Are they referring to above mentioned issue by myself earlier? I can't find issue/blocker attached to exact layout issue.